cli/conf.py

```diff
@@ -15,6 +15,64 @@
 )
 
 
+class ConfigWrapper(object):
+    """
+    The helper class for the InfraRed configuration.
+    """
+    _DEFAULT_SECTION = 'defaults'
+
+    def __init__(self, conf):
+        self.config = conf
+
+    def get_settings_dirs(self, section=_DEFAULT_SECTION):
+        """
+        Returns the list of the configured settings folders.
+        """
+        return self.config.get(section, 'settings').split(
+            os.pathsep)
+
+    def get_modules_dir(self, section=_DEFAULT_SECTION):
+        """
+        Gets the modules directory.
+        """
+        return self.config.get(section, 'modules')
+
+    def get_playbooks_dir(self, section=_DEFAULT_SECTION):
+        """
+        Gets the playbooks directory.
+        """
+        return self.config.get(section, 'playbooks')
+
+    def build_app_settings_dirs(self, app_name):
+        return [os.path.join(path, app_name)
+                for path in self.get_settings_dirs()]
+
+    def validate(self):
+        """
+        Validates the configuration.
+        """
+        # Validates at least one settings dir exists
+        dirs = self.get_settings_dirs()
+        if not any([os.path.exists(path)
+                   for path in dirs]):
```

Comment:
There's no reason for this line to be in a new line (it's not too long & it's less readable also)